src/libcollections/btree/node.rs

```diff
@@ -1359,6 +1359,9 @@ struct MoveTraversalImpl<K, V> {
     is_leaf: bool
 }
 
+unsafe impl<K: Send, V: Send> Send for MoveTraversalImpl<K, V> { }
+unsafe impl<K: Sync, V: Sync> Sync for MoveTraversalImpl<K, V> { }
```

Comment:
With the usage of `Unique`, I think these `unsafe impl` blocks can go away, right?